api/src/controllers/documents.ts

```diff
@@ -49,7 +49,40 @@ export let postDocuments = (req: Request, res: Response, next: NextFunction) =>
 }
 export let getDocuments = (req: Request, res: Response, next: NextFunction) => {
     try{
-        return res.status(200).send("Returned Documents")
+        const dummyData = [
```

Comment:
Here we define some dummy data which is a mock of a returned array of documents from Azure